README.md

```diff
@@ -801,15 +801,15 @@ See the [AUTHORS][] file for author contact information.
 License
 -------
 
-Bedrock and all Bedrock modules are:
+Bedrock core libraries are licensed under the [Apache-2.0](LICENSE).
```

Comment:
"... the Apache-2.0" needs fixing.  should it be "Apache 2.0 License" or a more official name:
```suggestion
Bedrock core libraries are licensed under the [Apache License, Version 2.0](LICENSE).
```

Reply:
I was sticking with the robot friendly SPDX moniker we use everywhere else, but happy to spell it out here.